app.js

```diff
@@ -13,6 +13,28 @@ function hasFilter(name, filters) {
   return false;
 }
 
+function getParameterByName(name) {
+  var urlParams = window.location.search;
+  name = name.replace(/[\[\]]/g, "\\$&");
+
+  var results = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)").exec(urlParams);
```

Comment:
No need for parsing url parameters, the `URL` constructor can be used instead:
```
var params = new URL(window.location).searchParams;
```
This gives you a ready to use object to retrieve the parameters. See https://developer.mozilla.org/en-US/docs/Web/API/URL

It seems to have decent browser support too except for Edge/IE, but I've never tested the site in these browsers.